src/AchForm.js

```diff
@@ -331,15 +349,27 @@ export default class AchForm extends Component {
     const onError = this.onError
     const onComplete = this.onComplete
     const onValidationError = this.onValidationError
-    account.saveWithSecureForm(
-      this.props.publicKey,
-      form,
-      {
-        onError,
-        onComplete,
-        onNext,
-        onValidationError,
-      })
+    if (model === modelTypes.ACCOUNT || !!model === false) {
+      account.saveWithSecureForm(
+        this.props.accessToken,
+        form,
+        {
+          onError,
+          onComplete,
+          onNext,
+          onValidationError,
+        })
+    } else {
+      instrument.saveWithSecureForm(
+        this.props.accessToken,
+        form,
+        {
+          onError,
+          onComplete,
+          onNext,
+          onValidationError,
+        })
+    }
```

Comment:
If we want to stay consistent with interfaces...

I recommend using the following pattern to make it easier to test:
```let modelToSave = instrument
if (model === modelTypes.ACCOUNT || !!model === false) {
  modelToSave = account
}

modelToSave.saveWithSecureForm(...)```

Reply:
I am a bit confused about why you are being shown code that has been removed. This interface should be consistent across ACH/CC and Plaid